configure.ac

```diff
@@ -17,7 +17,7 @@ RPM_RELEASE=0
 AC_SUBST(RPM_RELEASE)
 if test -d ".git" ; then
   AC_CHECK_PROG(GIT_CHECK, git, yes)
-  if test x"$GIT_CHECK" = x"yes"; then
+  if test x"$GIT_CHECK" = x"yes" -a x"$freebsd" != x"yes"; then
```

Comment:
Emmm, it will generate an error/warning. And I don't like those if I can prevent them.
Now the whole autobuild stuff is littered with warnings. (atleast when I run it under FreeBSD) so it is just another one of may.
That being said, FreeBSD does have a rpm-package:
  /usr/ports/archivers/rpm4
I do not have that installed, but I'm not sure what would happen if it was.
So the alternative is to add it to the dependancies. Then I'm not sure what happens if it start building a FreeBSD-Ceph-RPM 8-)
I would just leave it as is..